File: py/.py
import io
import os
import sys

# prepare stuff for faster IO
input = io.BytesIO(os.read(0, os.fstat(0).st_size)).readline
print = sys.stdout.write

# read line
s = input().decode("utf-8")

# parse string
nums = map(int, s.split(" "))

# print sum of nums
print(s.replace(" ", " + ") + " = " + str(sum(nums)) + "\n")

File: js/.js
// prepare stuff for faster IO
const fs = require("fs")

// read stdin
const s = fs.readFileSync(process.stdin.fd).toString()

// parse string
const nums = s.split(" ").map(Number)

// print sum of nums
process.stdout.write(
  s.replaceAll(" ", " + ") +
    " = " +
    nums.reduce((acc, curr) => acc + curr, 0) +
    "\n"
)
